guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>The <code>arcgis.learn</code> module includes the <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#timeseriesnetwork" target="_blank">TimeSeriesNetwork</a> class to train deep learning models on timeseries tabular data. Timeseries tabular data can be in the form of a <a href="https://developers.arcgis.com/python/guide/working-with-feature-layers-and-features/" target="_blank">feature layer</a>, <a href="https://developers.arcgis.com/python/guide/introduction-to-the-spatially-enabled-dataframe/" target="_blank">spatially enabled dataframe</a>, or simple dataframe. </p><p> </p><p><code>TimeSeriesNetwork</code> supports training models for both univariate and multivariate cases. </p><p> </p><p>In multivariate cases, explanatory variables can take the form of fields in the attribute table of the feature layer, raster datasets, or distance features used to calculate proximity values for use as additional variables. &nbsp;In univariate cases, the dependent variable can belong to the feature layer or raster. <code>TimeSeriesNetwork</code> can be used to predict a final raster from a series of rasters representing change over a period of time.  </p><p> </p><p>In addition to validation of model performance based on the training data, predictions can also produce either features or a prediction raster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>